src/Servers/IIS/IIS/test/Common.FunctionalTests/FrebTests.cs

```diff
@@ -90,10 +90,7 @@ public async Task CheckFailedRequestEvents()
         AssertFrebLogs(result, new FrebLogItem("ANCM_INPROC_ASYNC_COMPLETION_COMPLETION", "2"));
     }
 
-    // I think this test is flaky due to freb file not being created quickly enough.
-    // Adding extra logging, marking as flaky, and repeating should help
     [ConditionalFact]
-    [Repeat(10)]
```

Comment:
General note: `[Repeat]` is useful when testing reliability but just makes things _more_ flaky if checked in for a non-quarantined test.